src/main/java/com/lorepo/icf/utils/ExtendedRequestBuilder.java

```diff
@@ -10,9 +11,8 @@ public class ExtendedRequestBuilder extends RequestBuilder {
 	
 	private static boolean withCredentials = false;
 	private static String signingPrefix = null;
-	private static List<String> whitelist = new ArrayList<String>(); //contain trusted pages
-	private static String[] trustedPathsName = {"file/serve", "capacitor", "mauthor"};
```

Comment:
Sprawdzenie z "file/serve" przeniesiono do signURL.
"capacitor" usunięto, gdyż mLibro powinno zastosować nową metodę aby Android z nowym mechanizmem współpracował